src/app/modules/item/pages/item-log-view/item-log-view.component.ts

```diff
@@ -83,7 +83,7 @@ export class ItemLogViewComponent implements OnChanges, OnDestroy {
       {
         field: 'item.user',
         header: $localize`User`,
-        enabled: watchingGroupId && [ 'Chapter', 'Task', 'Course' ].includes(type),
+        enabled: watchingGroup && !watchingGroup.login,
```

Comment:
add a comment next it to explain what it means
```suggestion
        enabled: watchingGroup && !watchingGroup.login, // only if in watching mode and watch group is not a user
```